chef/data_bags/crowbar/template-crowbar.json

```diff
@@ -18,24 +18,37 @@
         "machine-install": { "password": "machine_password" },
         "crowbar": { "password": "crowbar" }
       },
-      "simple_proposal_ui": true
+      "simple_proposal_ui": true,
+      "upgrade": {
+        "db_dump_location" : "/var/lib/pgsql/db-before-cloud6.dump"
+      }
     }
   },
   "deployment": {
     "crowbar": {
-      "crowbar-revision": 10,
+      "crowbar-revision": 0,
       "crowbar-applied": false,
+      "schema-revision": 11,
       "element_states": {
-        "crowbar": [ "all" ]
+        "crowbar": [ "all" ],
+        "crowbar-upgrade": [ "readying", "ready", "applying", "crowbar_upgrade" ],
+        "crowbar-db-dump": [ "readying", "ready", "applying", "crowbar_upgrade" ]
```

Comment:
Oh, I almost forgot this. Why do you want to run these for anything else but "crowbar_upgrade" here. I know we need it for tex to be able to restart chef-client, but here I am pretty sure we only need "crowbar_upgrade".